x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_flyout_table/indicators_flyout_table.tsx

```diff
@@ -46,18 +23,39 @@ const search = {
 
 export interface IndicatorsFlyoutTableProps {
   indicator: Indicator;
+  fieldTypesMap: { [id: string]: string };
 }
 
-export const IndicatorsFlyoutTable: VFC<IndicatorsFlyoutTableProps> = ({ indicator }) => {
-  const items: Array<{ field: string; value: string }> = [];
-
-  for (const key in indicator.fields) {
-    if (!indicator.fields.hasOwnProperty(key)) continue;
-    items.push({
-      field: key,
-      value: unwrapValue(indicator, key as RawIndicatorFieldId) || EMPTY_VALUE,
-    });
-  }
+export const IndicatorsFlyoutTable: VFC<IndicatorsFlyoutTableProps> = ({
+  indicator,
+  fieldTypesMap,
+}) => {
+  const items: string[] = Object.keys(indicator.fields);
+  const columns = [
+    {
+      name: (
+        <FormattedMessage
+          id="xpack.threatIntelligence.indicator.flyoutTable.fieldColumnLabel"
+          defaultMessage="Field"
+        />
+      ),
+      render: (field: string) => field,
+    },
+    {
+      name: (
+        <FormattedMessage
+          id="xpack.threatIntelligence.indicator.flyoutTable.valueColumnLabel"
+          defaultMessage="Value"
+        />
+      ),
+      render: (field: string) =>
+        IndicatorField({
```

Comment:
I think it would be better to use jsx here `(<IndicatorField indicator="..." field="..." fieldTypesMap="...")` https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l

Reply:
awesome thanks for the link, reading it now!

Reply:
done